src/packages/missing.md

```diff
@@ -0,0 +1,22 @@
+# Missing packages
+
+Hybrid ABI and CheriABI package repositories might be missing third-party
+software due to the following reasons:
+* There is no FreeBSD port including that software or it is broken on AArch64;
+
+  Check [FreshPorts](https://www.freshports.org/),
+  [FreeBSD Wiki](https://wiki.freebsd.org/WantedPorts),
+  [FreeBSD Bugzilla](https://bugs.freebsd.org/bugzilla/) and
+  the [freebsd-ports mailing
+  list](https://lists.freebsd.org/subscription/freebsd-ports) to find out
+  if anyone is working on software you are interested in.
+
+* A CheriBSD port with the software is broken on Morello, does not respect a
+  custom local base (`/usr/local64`) when compiled for the hybrid ABI or is not
+  adapted to CheriABI.
```

Comment:
This can include other reasons due to the current state of CheriBSD, e.g. missing Linuxulator, and I wanted to explicitly mention CheriABI. I've updated the paragraph to mention CheriABI as an example case rather than a separate one.